ui/src/message_popup/experience_page.jsx

```diff
@@ -168,25 +157,27 @@ export default React.createClass({
   },
 
   renderInstructions() {
+    const {scaffolding, gameSession} = this.state;
     return (
-      <InstructionsCard 
-        sessionLength={this.state.sessionLength}
-        onStartPressed={this.onStartPressed}
-        email={this.state.email}
-        itemsToShow={this.props.query}
-        helpType={this.state.helpType}
-        />);
+      <VelocityTransitionGroup enter={{animation: "callout.pulse", duration: 500}} leave={{animation: "slideUp"}} runOnMount={true}>
+        <div>
+          <InstructionsCard 
+           itemsToShow={this.props.query}
+           />
+          <ScaffoldingCard
+            email={gameSession.email}
```

Comment:
Let's have the `ScaffoldingCard can own the email state, and not handing that information back to the experience page until it's wrapped up into a `gameSession`.

So this can become `initialEmail` to indicate it's like a "default value" but `ScaffoldingCard` tracks its own state.